app/views/organisations/new.html.erb

```diff
@@ -6,16 +6,35 @@
 <%= form_for @organisation, url: organisations_path do |form| %>
   <div class="govuk-form-group <%= field_error(@organisation, "name") %>">
     <%= form.label :name, "Organisation name", class: "govuk-label" %>
-    <div class="govuk-hint">If your organisation name does not appear below <%= link_to "contact us", technical_support_new_help_path, class: "govuk-link" %></div>
+    <div class="govuk-hint">If your organisation name does not appear
+      below <%= link_to "contact us", technical_support_new_help_path, class: "govuk-link" %></div>
```

Comment:
Not sure where we officially stand on line length, but does this need to be broken on two?

Reply:
Done, and at line 29.